doc/TUTORIAL.md

```diff
@@ -148,7 +148,7 @@ Description of the code:
    - `pattern` defines the variable part of the route, everything that comes after `/example/`. It can include any
      number of named parameters. These are converted into
      regular expressions by [`path-to-regexp`][path-to-regexp]. the list of the valid categories can be seen
-6. `route()` declares the URL path at which the service operates. It also maps components of the URL path to handler parameters.
+6. `handle()` declares the URL path at which the service operates. It also maps components of the URL path to handler parameters.
```

Comment:
Should this item in the list be removed instead? It looks like items 5 and 6 are duplicates.

Reply:
Well, we still need a text description for number 6 in the code. Maybe replace 6 with 8, and merge 5 and 7 together?

Reply:
Ah, right, these numbers correspond with the code comments. Yea, what you're suggesting makes sense.

Reply:
Yes the description for the `handle()` function should be:

> All badges must implement the `async handle()` function that receives parameters to render the badge...

and the description for `route()` should be

> `route()` declares the URL path at which the service operates. It also maps components of the URL path to handler parameters.

its the numbering that has become mixed up.